File: kotlin-ide/j2k/old/testData/fileOrElement/issues/kt-809.java
//file
package demo;

class Container {
  int myInt = 1;
}

class One {
  static Container myContainer = new Container();
}

class IntContainer {
  IntContainer(int i) {}
}

class Test {
  void putInt(int i) { }
  void test() {
    putInt(One.myContainer.myInt);
    new IntContainer(One.myContainer.myInt);
  }
}

File: j2k/testData/fileOrElement/issues/kt-809.kt
package demo

internal class Container {
    internal var myInt = 1
}

internal object One {
    internal var myContainer = Container()
}

internal class IntContainer internal constructor(i: Int)

internal class Test {
    internal fun putInt(i: Int) {
    }

    internal fun test() {
        putInt(One.myContainer.myInt)
        IntContainer(One.myContainer.myInt)
    }
}